src/Components/Catalog/Catalog.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import PropTypes from "prop-types"
+import Item from "./Item"
+
+export default class Catalog extends Component {

```

Comment:
make this a component function. we dont need a component class for a simple UI component.